lib/content-services/src/lib/permission-manager/components/permission-list/permission-list.component.scss

```diff
@@ -92,3 +85,12 @@
         box-shadow: 0 8px 9px -5px var(--adf-theme-foreground-divider-color), 0 15px 22px 2px var(--adf-theme-foreground-divider-color);
     }
 }
+
+.adf-datatable .adf-datatable-row .adf-datatable-cell-value .adf-permission-role-column-header {
+    position: relative;
+    height: 40px;
```

Comment:
Height style seems not working because of some reasons. If you check console it is grayed out:
<img width="395" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/c9f20f1a-88a8-491b-915c-737feb1d908d">